stdlib/public/core/Flatten.swift.gyb

```diff
@@ -313,40 +270,50 @@ public struct ${Collection}<
   /// reachable from `startIndex` by zero or more applications of
   /// `successor()`.
   public var endIndex: Index {
-    return ${Index}(_endIndexOfFlattened: _base)
+    return ${Index}(_base.endIndex, nil)
   }
```

Comment:
It could be an advantage to define it as a constant -- what kind of representation do you have in mind?